stable/graylog/values.yaml

```diff
@@ -240,6 +240,9 @@ graylog:
     ## If true, Maxmind GeoLite2 will be installed to ${GRAYLOG_HOME}/geoip location
     ##
     enabled: false
+    ## If true, ddbbUri points to the URI where to find the DDBB
+    # https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/
+    ddbbUri: ""
```

Comment:
We need a default URI to `https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz`
Otherwise, chart will not working when you enable Geoip.

Reply:
The problem is that there is no default DB available any more, it is not publicly available... So someone would need to put it somewhere and keep it updated.

Reply:
@titansmc ok, then please just update the name from `ddbb` to `mmdb`. I think `mmdb` Maxmind Database is more clear on this parameter.

Reply:
@KongZ done. Since this is my first PR, I might be doing something wrong, sorry.

Reply:
Hi @KongZ  I just realized that the README.md was outdated and I just pushed the changes to my branch, should I request another PR ?

Reply:
@titansmc yes, you have to create another PR. Thanks :)